v2/component/launcher.go

```diff
@@ -427,6 +430,9 @@ func (l *Launcher) RunComponent(ctx context.Context, cmdArgs []string) error {
 		})
 
 		rtoa, ok := l.runtimeInfo.OutputArtifacts[name]
+		if !filepath.IsAbs(rtoa.MetadataPath) {
+			return metadataErr(fmt.Errorf("unexpected output artifact metadata file %q: must be absolute local path", rtoa.MetadataPath))
```

Comment:
When does this happen btw?

Reply:
I reproduced this problem when running https://github.com/amygdala/code-snippets/blob/master/ml/vertex_pipelines/pytorch/cifar/pytorch_cifar10_vertex_pipelines.ipynb in v2 compatible mode. I don't understand why that happened. (There are more details in chat group)